react/components/NotificationBar/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { memo } from 'react'
+import { bool, string, oneOf } from 'prop-types'
+import classNames from 'classnames'
+
+import styles from './styles.css'
+
+const NotificationBar = ({
```

Comment:
not entirely sure about the name `NotificationBar` (not against it, just let me think about it for a moment)

Reply:
I agree, looks strange to me too.
But I made a research and seems this name is the most used.
eg. https://www.google.com/search?q=notification+bar+magento&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjYz82KjKXhAhXfIbkGHRTaDPMQ_AUIDigB&biw=2560&bih=1338